lib/logstash/outputs/kafka.rb

```diff
@@ -106,6 +106,8 @@ class LogStash::Outputs::Kafka < LogStash::Outputs::Base
   config :max_request_size, :validate => :number, :default => 1_048_576 # (1MB) Kafka default
   # The key for the message
   config :message_key, :validate => :string
+  # Headers added to kafka message in the form of key-value pairs
+  config :message_headers, :validate => :hash
```

Comment:
Having a default value would be beneficial to avoid always `nil?` checks.
This also requires doc changes:
- add to the [params table](https://github.com/logstash-plugins/logstash-integration-kafka/blob/main/docs/input-kafka.asciidoc?plain=1#L97) with alphabetical order
- explain it in the main doc content (obeys the order of the param options table)

Reply:
Done